text-frontend/__main__.py

```diff
@@ -11,6 +11,31 @@
 # debug constants
 USER = {"id": "1234", "display_name": "John Doe", "auth_method": "local"}
 
+VALID_LABELS = [
```

Comment:
eventually we should probably send the valid labels in the task to the client

Reply:
That's a good idea. Could we maybe also add a couple of positive labels?

Reply:
ok, I see we have at least ... "creative", "beautiful",  "informative",

Reply:
I used the same list which was already present in the `TextLabels` enum in `protocol.py`. Suggest we move any changes to the labels list to a separate issue/PR to avoid this one getting too broad

Reply:
With the latest changes the backend should be sending the list of valid labels as part of the task. I still need to test to verify this works